core/src/test/scala/caliban/validation/ValidationSchemaSpec.scala

```diff
@@ -13,12 +13,12 @@ object ValidationSchemaSpec
       def check(gql: GraphQL[Any], expectedMessage: String): IO[ValidationError, TestResult] =
         assertM(gql.interpreter.run, fails[ValidationError](hasField("msg", _.msg, equalTo(expectedMessage))))
 
-      suite("ValidationSchemaSpec")({
+      suite("ValidationSchemaSpec")(
         suite("InputObjects")(
           testM("name can't start with '__'") {
             check(
               graphQL(resolverWrongMutationUnderscore),
-              "InputObject can't start with '__': __name"
+              "input value in InputObject can't start with '__': __name"
```

Comment:
I’m nitpicking but can you make the first letter uppercase, to be consistent with the rest?

Reply:
what is your opinion on the `nameExtractor` and `messageBuilder` stuff? I could do the same for this validation with overloaded methods one for `__Field` and one for `__InputValue`

Reply:
I don't have a strong opinion on it, you can do as you feel. As long as the result message looks good 😄

Reply:
I improved the error messages to also provide the context where the validation error occurred
The interface and input object validation error messages follow the same schema now and lead to cleaner code